tests/handlers/test_upload_api.py

```diff
@@ -8,21 +8,20 @@
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
-from shutil import rmtree
-import tempfile
 import re
+import tempfile
+from shutil import rmtree
 
 from preggy import expect
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)